.github/workflows/publish_ci.yml

```
@@ -67,3 +67,12 @@ jobs:
       run: |
         cd src/FSharp.Data.GraphQL.Server.Middleware
         dotnet nuget push ../../nuget/*Server.Middleware*.nupkg -s "github" -k ${{secrets.GITHUB_TOKEN}}
+
+    - name: Pack FSharp.Data.GraphQL.Relay project
+      run: |
+        cd src/FSharp.Data.GraphQL.Relay
+        dotnet pack --no-build --nologo --configuration Release /p:IsNuget=true -o ../../nuget
+    - name: Publish FSharp.Data.GraphQL.Relay project to GitHub
+      run: |
+        cd src/FSharp.Data.GraphQL.Relay
+        dotnet nuget push ../../nuget/*Relay*.nupkg -s "github" -k ${{secrets.GITHUB_TOKEN}}
```

Project name must be `FSharp.Data.GraphQL.Server.Relay`
```suggestion
    - name: Pack FSharp.Data.GraphQL.Server.Relay project
      run: |
        cd src/FSharp.Data.GraphQL.Server.Relay
        dotnet pack --no-build --nologo --configuration Release /p:IsNuget=true -o ../../nuget
    - name: Publish FSharp.Data.GraphQL.Server.Relay project to GitHub
      run: |
        cd src/FSharp.Data.GraphQL.Relay
        dotnet nuget push ../../nuget/*Server.Relay*.nupkg -s "github" -k ${{secrets.GITHUB_TOKEN}}
```